spec.html

```diff
@@ -20463,23 +20470,17 @@ <h1>Runtime Semantics: IteratorBindingInitialization</h1>
       <h1>Runtime Semantics: EvaluateBody</h1>
       <p>With parameters _functionObject_ and List _argumentsList_.</p>
       <emu-grammar>
-        AsyncConciseBody : AssignmentExpression
+        AsyncConciseBody : ExpressionBody
       </emu-grammar>
       <emu-alg>
         1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
         1. Let _declResult_ be FunctionDeclarationInstantiation(_functionObject_, _argumentsList_).
         1. If _declResult_ is not an abrupt completion, then
-          1. Perform ! AsyncFunctionStart(_promiseCapability_, |AssignmentExpression|).
+          1. Perform ! AsyncFunctionStart(_promiseCapability_, |ExpressionBody|).
         1. Else,
           1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; _declResult_.[[Value]] &raquo;).
         1. Return Completion { [[Type]]: ~return~, [[Value]]: _promiseCapability_.[[Promise]], [[Target]]: ~empty~ }.
       </emu-alg>
-      <emu-grammar>
-        AsyncConciseBody : `{` AsyncFunctionBody `}`
-      </emu-grammar>
-      <emu-alg>
-        1. Return the result of EvaluateBody of |AsyncFunctionBody| passing _functionObject_ and _argumentsList_ as the arguments.
-      </emu-alg>
```

Comment:
Musing: I wonder if we should have ecmarkup auto-generate chain production expansions somehow so the reader can get a clickable link for the RHS.

Reply:
Theoretically possible, but I think it would go substantially beyond ecmarkup's current capabilities just to deduce the chain rules that have been left implicit.